WordPress/WordPressShare/PostStatusPickerViewController.swift

```diff
@@ -0,0 +1,98 @@
+import Foundation
+import UIKit
+import WordPressShared
+
+class PostStatusPickerViewController : UITableViewController
+{
+    // MARK: - View Lifecycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupView()
+        setupTableView()
+        setupNoResultsView()
+        loadStatuses()
+    }
+    
+    
+    // MARK: - UITableView Methods
+    override func numberOfSectionsInTableView(tableView: UITableView) -> Int {
+        return 1
+    }
+    
+    override func tableView(tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return statuses.count
+    }
+    
+    override func tableView(tableView: UITableView, heightForRowAtIndexPath indexPath: NSIndexPath) -> CGFloat {
+        return rowHeight
+    }
+    
+    override func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
+        var cell = tableView.dequeueReusableCellWithIdentifier(reuseIdentifier)
```

Comment:
> Maybe we could/should detach WPTableViewCellSubtitle from ImmuTable instead 

Definitely. `WPReusableTableViewCell` and the four system cells (Default, Value1, Value2, Subtitle) aren't really tied to ImmuTable (other than being in that file), and they're just about `UITableViewCell` reusability.